public/api/conf/1.0/docs/testing.md

```diff
@@ -0,0 +1,3 @@
+You can use the sandbox environment to [test this API](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing).
+
+It currently only supports [stateless behaviour](https://developer.service.hmrc.gov.uk/api-documentation/docs/testing/stateful-behaviour). 
```

Comment:
"stateless behaviour" and "stateful-behaviour"
Maybe something wrong ?

Reply:
No, that page covers both scenarios: https://developer.service.hmrc.gov.uk/api-documentation/docs/testing/test-users-test-data-stateful-behaviour